docs/kitsune-support.md

```diff
@@ -4,8 +4,8 @@
 
 High level:
 
-- [SUMO Infra home](https://github.com/mozmeao/infra/tree/master/apps/sumo)
-- [Deploying SUMO](https://github.com/mozilla/kitsune/tree/master/k8s#deploying-sumo)
+- [SUMO Infra home](https://github.com/mozilla-it/sumo-infra)
+- [Deploying SUMO](https://github.com/mozilla-it/kitsune/tree/master/k8s#deploying-sumo)
```

Comment:
Since we are using this repo, I think it would be easier to have everything related to the app referenced here.

Reply:
Pointed it to this repo now, thanks!